### ISSUE TITLE
Store config disappears when selecting website scope

### ISSUE THREAD

**Issue by erfanimani** [magestat/magento2-facebook-pixel#16]
**To Reproduce**

* As an admin, navigate to Magestat Facebook pixel
* Select website scope

**Expected behavior**
* Allow config to be set on a website scope

**Actual behavior**
* Magestat config disappears

**Comment by williankeller** [comment]
This is planned for one of our next releases, 1.0.6 or 1.0.7

**Comment by seansan** [comment]
Also waiting for this (we use 8 storeviews! so I am reading this as: multistore is not supported?) ... thanks all!

**Comment by erfanimani** [comment]
Yeah, at the moment it doesn't look like it's support. Looking forward to
the new release. Thanks!

**Comment by erfanimani** [comment]
Hi there — is there an update on this?

**Comment by erfanimani** [comment]
@williankeller I can probably work on this at some point, if you want

**Comment by williankeller** [comment]
That would be lovely, @erfanimani!
